rclcpp/include/rclcpp/create_subscription.hpp

```diff
@@ -64,10 +74,46 @@ create_subscription(
   using rclcpp::node_interfaces::get_node_topics_interface;
   auto node_topics = get_node_topics_interface(std::forward<NodeT>(node));
 
+  auto node_timer_interface = node_topics->get_node_timers_interface();
```

Comment:
nitpick: the scope of this variable can be reduced to only be within in the if statement's true block.

Reply:
Good point, will apply.